content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -144,6 +146,52 @@ $ spin cloud kv list
 
 Now using these commands you can see your Spin applications, their respective labels and their connected databases and stores.
 
+### Linking Resources Between Applications
+
+Linking SQLite and Key Value resources between applications in Fermyon Cloud is possible. Here is an example of how you would link (share) a Key Value resource between two applications in Fermyon Cloud. Let's start by creating a new Key Value resource:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin cloud key-value create my-shared-kv
+Key value store  "my-shared-kv" created
```

Comment:
Is the extra space after `store` there in the console output?  If so we should remove that...

Reply:
Yes, the extra space is present on the `create` subcommand. 
```
Key value store  "my-shared-kv-test" created
```

Reply:
https://github.com/fermyon/cloud-plugin/pull/197/files